### PR TITLE
Graphql cleanups II: the next generation

### PR DESCRIPTION
**Pull request by jauco**

### CODE REVIEW

File: timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/stores/BdbTripleStore.java [hunk 1]
```diff
@@ -127,21 +129,29 @@ public void setPrefix(String cursor, String prefix, String iri) throws RdfProces
...
+    }
   }
 
   @Override
   public void addValue(String cursor, String subject, String predicate, String value, String dataType, String graph)
       throws RdfProcessingFailedException {
     try {
       put(subject + "\n" +
-        predicate,
+        predicate + "\n" + Direction.OUT.name(),
```
Comment (MMaas3) [inline timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/stores/BdbTripleStore.java]: Why is there a new line after predicate?
Before the predicate and dataType were on one line after the change the are on separate lines and the direction and the dataType are on one line. I would expect one line with all the predicate infromation

File: timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/datafetchers/PaginationHelper.java [hunk 2]
```diff
@@ -36,8 +42,8 @@
       );
     } else {
       return PaginatedList.create(
-        cursors[0],
-        cursors[1],
+        startedFromCursor ? cursors[0] : null,
```
Comment (MMaas3) [inline timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/datafetchers/PaginationHelper.java]: Cursors will benefit from a more descriptive name like "prevCursor" and "nextCursor"

File: timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/serializable/serializations/base/CollectionsOfEntitiesSerializationTest.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    // Map<String, CollectionsOfEntitiesSerialization.AggregatedEntity> persons = sut
+    //   .allEntities.get("http://example.com/Person");
+    // List person1BList = persons.get("http://example.com/1").relations.get("b");
+    // List person2BList = persons.get("http://example.com/2").relations.get("b");
+    // assertThat(person1BList.get(0), sameInstance(person2BList.get(0)));
+    // assertThat(person1BList.get(1), sameInstance(person2BList.get(1)));
+  }
+
+
+  class TestCollectionsOfEntitiesSerialization extends CollectionsOfEntitiesSerialization {
```
Comment (MMaas3) [inline timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/serializable/serializations/base/CollectionsOfEntitiesSerializationTest.java]: Test should be removed or (re)implemented.

Comment (jauco) [inline timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/serializable/serializations/base/CollectionsOfEntitiesSerializationTest.java]: Good catch

File: timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/stores/BdbTripleStore.java (hunk 1 above)
Comment (jauco) [inline timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/stores/BdbTripleStore.java]: You're missing a comma. The key was `subject\npredicate` and is now `subject\npredicate\ndirection`

Comment (MMaas3) [inline timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/v5/bdbdatafetchers/stores/BdbTripleStore.java]: I see now. :)

**Pull request merged**